### ISSUE TITLE
raspberrypi0-2w-64 only 284MB ram and "vc4-fkms-v3d" dtoverlay issue

### ISSUE THREAD

**Issue by ClintHaerinck** [balena-os/balena-raspberrypi#812]
balenaOS  2.94.4 (production)
raspberrypi0-2w-64

My device only shows 284MB available RAM in balenaCloud.
Also when applying device tree overlay vc4-fkms-v3d, the device doesn't boot anymore.

**Comment by floion** [comment]
Hi, is this OS built by you?

**Comment by ClintHaerinck** [comment]
@floion No, downloaded from balenaCloud. It's also strange no revision number is added to the version.
